2/contrib/s2i/run

```diff
@@ -60,6 +60,18 @@ function create_jenkins_config_from_templates() {
 }
 
 function install_plugins() {
+  mkdir -p ${JENKINS_HOME}/plugins
+    
+  if [ "$(ls -A /usr/lib/jenkins/*.hpi 2>/dev/null)" ]; then
+      # found rhel rpm installed plugins in the 3.7 or later location
+      echo "Copying $(ls /usr/lib/jenkins/*.hpi | wc -l) RHEL RPM installed Jenkins plugins to ${JENKINS_HOME} ..."
+      for FILENAME in /usr/lib/jenkins/*hpi ; do cp $FILENAME ${JENKINS_HOME}/plugins/`basename $FILENAME .hpi`.jpi; done
+  elif [ "$(ls -A /usr/lib64/jenkins/*.hpi 2>/dev/null)" ]; then
```

Comment:
i don't think this should be an elif, we should just do both.

Reply:
ok, if both though, I'm giving the newer flavor precedence (i.e. executed afterward to override whatever was done previously)

Reply:
what are we considering the "new" flavor?  lib64?

Reply:
old == lib64, new == lib

Reply:
ok. I guess i don't particularly care either way.